mmv1/third_party/terraform/website/docs/d/compute_networks.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+subcategory: "Compute Engine"
+description: |-
+  List networks in a Google Cloud project.
+---
+
+# google\_compute\_networks
+
+List all networks in a specified Google Cloud project.
+
+## Example Usage
+
+```tf
+data "google_compute_networks" "my-networks" {
+  project = "my-cloud-project"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Required) The name of the project.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following attributes are exported:
+
+* `id` - an identifier for the resource with format projects/{{project}}/global/networks/
+
+* `description` - Description of this network.
+
+* `networks` - The list of networks in the specified project.
+
+* `project` - The project name being queried.
```

Comment:
`project` is already mentioned in `Argument Reference`

Reply:
Should I leave it as part of the return or remove it? I figured returning it would at least verify which project the data source was pulling from.

Reply:
I see your point. It is good to return it.